README.md

```diff
@@ -1,2 +1,110 @@
-# recipe_app_rails
-Good repo
+# Blog App
+
+> The recipe is created with ruby on rails and allows logged in users to create recipes from food items they have available. It also allows them to view public recipes shared by other users.
+
+## Project screenshots
+
+![screenshot()]()
```

Comment:
- [ ] It would be better if you provide a screenshots of your app because it makes easy for users to imagine before using your app.